Patterns/Portuguese/Portuguese-Numbers.yaml

```diff
@@ -31,9 +31,6 @@ AllIntRegex: !nestedRegex
 AllPointRegex: !nestedRegex
   def: ((\s+{ZeroToNineIntegerRegex})+|(\s+{AllIntRegex}))
   references: [ ZeroToNineIntegerRegex, AllIntRegex ]
-AllFloatRegex: !nestedRegex
```

Comment:
Why was this removed from the YAML file? Shouldn't it just be moved up?

Reply:
This Regex was removed because it was duplicated. In the generated NumbersDefinitions for C# this line didn't present any problem because the duplicated regex is ignored, but for Javascript it was neccesary to delete it.

Reply:
Sorry, I didn't realize that.